apps/els_core/src/els_utils.erl

```diff
@@ -453,15 +453,18 @@ compose_node_name(Name, Type) ->
             true ->
                 Name;
             _ ->
-                {ok, HostName} = inet:gethostname(),
+                HostName = els_config_runtime:get_hostname(),
```

Comment:
Replaced the `inet:gethostname()` with `els_config_runtime:get_hostname()` in all the relevant places in the code base.